src/reducers/widgets/index.js

```diff
@@ -11,17 +11,17 @@ const updateWidget = ( state, action, type, updater ) => {
 
   return [
     ...state.slice( 0, index ),
-       updater( state[ index ], action ),
+    updater( state[ index ], action ),
     ...state.slice( index + 1 )
   ]
 }
 
 const widgetReducer = ( state = [dummyClock], action ) => {
   switch( action.type ) {
-    case CLOCK_TICK:
-      return updateWidget( state, action, 'Clock', clockTicked )
-    default:
-      return state
+  case CLOCK_TICK:
```

Comment:
I prefer case statements indented under their enclosing switch - can we find the appropriate linting rule for that? (Or just undo this change)